Conventional/Convention.Cecil.cs

```diff
@@ -6,9 +6,9 @@ namespace Conventional
 {
     public static partial class Convention
     {
-        public static MustNotUseDateTimeNowConventionSpecification MustNotUseDateTimeNow

```

Comment:
The rename suggested below to "MustNotUseDateTimeDirectlyConventionSpecification" would need to be reflected here too.